website_sale_delivery_date/readme/USAGE.md

```diff
@@ -0,0 +1,5 @@
+This module extends the Odoo e-commerce checkout flow by adding a delivery date selector after the customer chooses a delivery method. It is designed for companies that want to allow their customers to specify a desired delivery date when placing an order online.
```

Comment:
How is this related to usage? Besides the same text is already in the DESCRIPTION section.